README.md

```diff
@@ -39,26 +43,31 @@ Documented 1 public library in 17.9 seconds
 Success! Docs generated into <path to dartdoc>/doc/api
 ```
 
-By default, the documentation is generated to the `doc/api` directory as static
-HTML files.
+By default, the documentation is generated to the `doc/api` directory as
+static HTML files.
+
+To view the generated documentation, you must load them with an HTTP server.
+To learn more, follow the [Viewing docs](#viewing-docs) guide.
 
 Run `dart help doc` to see the available command-line options.
 
 ## Viewing docs
 
-You can view the generated docs directly from the file system, but if you want
-to use the search function, you must load them with an HTTP server.
+To enable navigation and search, the generated docs must be
+served with an HTTP server.
 
-An easy way to run an HTTP server locally is to use the `dhttpd` package. For
-example:
+An easy way to run an HTTP server locally is to use [`package:dhttpd`][].
+For example:
 
 ```
 $ dart pub global activate dhttpd
-$ dhttpd --path doc/api
+$ dart pub global run dhttpd --path doc/api
```

Comment:
I made this change as I notice a lot of developers end up not adding the directory to their path. This is more likely to work :)